### DISCUSSION TITLE
wiki/heartmate/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#48] [Announcements]
# wiki/heartmate/

HeartMate 专为那些没有智能手环或手表却需要测量心率的用户而设计，可以方便快捷的测量和记录心率。使用 HeartMate 心率助手，你可以在没有智能手环等心率监测设备的情况下随时检测并记录自己的心率。 通过设置tag标签，快速记录当前状态，如饭后、运动后等，方便以后检索具有同样标签的心率数据，从而分析这些情景下的心率数据，发现健康问题。

https://xaoxuu.com/wiki/heartmate/

**Comment by fenychn0206** [comment]
呵呵
